File: startingPoint/02-EstruturaSequencial/Exercicios_Propostos/02.py
#!/usr/bin/python
# coding: latin-1

# 02 - Faça um programa que receba três notas e seus respectivos pesos, calcule e mostre a média ponderada dessas notas

nota1 = int(input("Digite a primeira nota: "))
peso1 = int(input("Digite o peso da primeira nota: "))

nota2 = int(input("Digite a segunta nota: "))
peso2 = int(input("Digite o peso da segunda nota: "))

nota3 = int(input("Digite a terceira nota: "))
peso3 = int(input("Digite o peso da terceira nota: "))

soma1 = nota1 * peso1
soma2 = nota2 * peso2
soma3 = nota3 * peso3
total = peso1 + peso2 + peso3
media = (soma1 + soma2 + soma3) / total

print(media)

File: SRC - PERL/00 - Estruturas Sequenciais/02.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faça um programa que receba três notas e seus respectivos pesos, calcule e mostre a média ponderada dessas notas

print("Digite a nota 1 > ");
my $nota1 = <>;

print("Digite o peso da nota 1 > ");
my $peso1 = <>;

print("Digite a nota 2 > ");
my $nota2 = <>;

print("Digite o peso da nota 2 > ");
my $peso2 = <>;

print("Digite a nota 3 > ");
my $nota3 = <>;

print("Digite o peso da nota 3 > ");
my $peso3 = <>;

my $soma1 = $nota1 * $peso1;
my $soma2 = $nota2 * $peso2;
my $soma3 = $nota3 * $peso3;

my $total = $peso1 + $peso2 + $peso3;

my $media = ($soma1 + $soma2 + $soma3) / $total;

say "A média ponderada das notas é ", $media;
